src/follower.py

```diff
@@ -87,21 +94,56 @@ def stop(self):
 
 
     """
-    Turn the wheels towards the last recognized object.
+    Turn the wheels and pans the camera towards the last recognized object.
     """
     def turn(self):
         self.convert_camera_angle()
-        self._fw.turn(self._turn_angle)
+        self.pan_camera()
 
+        if self.STRAIGHT_ANGLE - self.DEADZONE_WHEEL < self._turn_angle:
```

Comment:
Add comments for this if block that says which condition makes the vehicle turn left or right so it's easier to follow.

Reply:
Added comments